tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -405,28 +416,25 @@ static int test_wdt_bad_window_max(void)
 
 	return TC_FAIL;
 }
+#endif
 
+#if TEST_WDT_WAIT_MODE
 static int test_wdt_enable_wait_mode(void)
 {
-#ifndef CONFIG_PM
-	TC_PRINT("Testcase: %s\n", __func__);
-	ztest_test_skip();
-	m_state = WDT_TEST_STATE_IDLE;
-	return TC_SKIP;
-#else
```

Comment:
why remove this?

Reply:
what do you mean wrapped? what is the relation with NOINIT_SECTION?

Reply:
The statements after `ztest_test_skip` in the testcase will not be performed.

Reply:
It was kept. Please see line 147, 421, 507.
m_state will be updated in 
```c
if (m_testcase_index > 4) {
	m_state = WDT_TEST_STATE_IDLE;
}
```